DataDefinitions/Material.cs

```diff
@@ -102,6 +102,9 @@ static Material()
             var Tg_ShipSystemData = new Material("tg_shipsystemsdata", Data, Rare);
             var Tg_ResidueData = new Material("tg_residuedata", Data, Rare);
             var UnknownWakeScan = new Material("unknownwakedata", Data, Rare);
+            var Guardian_VesselBlueprint = new Material("guardian_vesselblueprint", Data, Rare);
```

Comment:
I may be wrong, but I don't think that `Guardian_VesselBlueprint` appears in Howard's spreadsheet.
Per the in-game HUD display, all 3 of these are very rare. But my capacity for Guardian Vessel prints is 100 while my capacity for the other two is 150. Thus, I think we should classify Guardian Vessel prints as "Very Rare" (with an auto-generated max capacity of 100 units) while classifying the other two as "Rare" (with an auto-generated max capacity of 150 units).

Reply:
I’ll check on Guardian_VesselBlueprint and conform the rarities and capacities.

Reply:
I agree. Done.